test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,29 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+
+    @Test
+    public void someElementsAreNull() throws Exception {
+        // only one null
+        assertAnyNull((Object) null);
+
+        // some objects null
+        assertAnyNull("abc", null, 123);
+        assertAnyNull(null, 321, null);
+
+        // all null objects
+        assertAnyNull((Object) null, (Object) null, (Object) null);
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertAnyNull(Object... objects) {
```

Comment:
You can consider just using `assertTrue.(Utils.isAnyNull(...))` (and also `assertFalse`) in the method on Line 41 instead of abstracting this out as a method. (`...` contains what you want to test.)

The reason why existing tests do that (`assertAreUnique` and `assertNotUnique`) is to avoid repeating usage of `Arrays.asList`.